olm-catalog/serverless-operator/hack/update-manifests.sh

```diff
@@ -15,9 +15,13 @@ target_serving_file="$target_dir/operator_v1alpha1_knativeserving_crd.yaml"
 target_eventing_file="$target_dir/operator_v1alpha1_knativeeventing_crd.yaml"
 rm -rf "$target_serving_file" "$target_eventing_file"
 
-serving_url="https://raw.githubusercontent.com/knative/operator/$version/config/300-serving.yaml"
-eventing_url="https://raw.githubusercontent.com/knative/operator/$version/config/300-eventing.yaml"
-
+if [[ "$version" == "1.0.1" || "$version" == "1.0.0" ]]; then
+  serving_url="https://raw.githubusercontent.com/knative/operator/knative-v1.0.1/config/300-serving.yaml"
+  eventing_url="https://raw.githubusercontent.com/knative/operator/knative-v1.0.0/config/300-eventing.yaml"
+else
+  serving_url="https://raw.githubusercontent.com/knative/operator/$version/config/300-serving.yaml"
+  eventing_url="https://raw.githubusercontent.com/knative/operator/$version/config/300-eventing.yaml"
+fi
```

Comment:
Can we use `knative-v1.0.1` for both eventing and serving so that we can remove this special-casing?

Reply:
Yes we can. This is the release of the **operator** not of either serving or eventing. So just using one version is sufficient here.

Reply:
```suggestion
serving_url="https://raw.githubusercontent.com/knative/operator/knative-$version/config/300-serving.yaml"
eventing_url="https://raw.githubusercontent.com/knative/operator/knative-$version/config/300-eventing.yaml"
```

Reply:
Sure I was hoping for that. If things are ok for Eventing no problem.

Reply:
Technically though we want to have an if there because someone may set `OPERATOR_VERSION` to some other value.

Reply:
I suspect it is going to be 1.0.1 so I am updating the urls.